lib/string-checker.js

```diff
@@ -130,7 +130,12 @@ StringChecker.prototype = {
 
         this._configuredRules.forEach(function(rule) {
             errors.setCurrentRule(rule.getOptionName());
-            rule.check(file, errors);
+
+            try {
+                rule.check(file, errors);
+            } catch (e) {
+                errors.add(e, 1, 0);
```

Comment:
Can we try to make "warnings" mechanism to report internal issues without failing code style check?

This PR is not critical anyway. User now gets a huge amount of fun, and to reduce it we need to hardcode somewhere a try-catch.